static/app/views/performance/transactionSummary/transactionSpans/spanSummary/spanSummaryCharts.tsx

```diff
@@ -0,0 +1,177 @@
+import {t} from 'sentry/locale';
+import type {Series} from 'sentry/types/echarts';
+import EventView, {type MetaType} from 'sentry/utils/discover/eventView';
+import {RateUnit} from 'sentry/utils/discover/fields';
+import {
+  type DiscoverQueryProps,
+  useGenericDiscoverQuery,
+} from 'sentry/utils/discover/genericDiscoverQuery';
+import {formatRate} from 'sentry/utils/formatters';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useParams} from 'sentry/utils/useParams';
+import {
+  AVG_COLOR,
+  THROUGHPUT_COLOR,
+  TXN_THROUGHPUT_COLOR,
+} from 'sentry/views/starfish/colors';
+import Chart, {ChartType} from 'sentry/views/starfish/components/chart';
+import ChartPanel from 'sentry/views/starfish/components/chartPanel';
+import {useSpanMetricsSeries} from 'sentry/views/starfish/queries/useSpanMetricsSeries';
+import {
+  SpanMetricsField,
+  type SpanMetricsQueryFilters,
+} from 'sentry/views/starfish/types';
+import {Block, BlockContainer} from 'sentry/views/starfish/views/spanSummaryPage/block';
+
+function SpanSummaryCharts() {
+  const organization = useOrganization();
+  const {spanSlug} = useParams();
+  const [spanOp, groupId] = spanSlug.split(':');
+
+  const location = useLocation();
+  const {transaction} = location.query;
+
+  const filters: SpanMetricsQueryFilters = {
+    'span.group': groupId,
+    'span.op': spanOp,
+    transaction: transaction as string,
+  };
+
+  const {
+    isLoading: isThroughputDataLoading,
+    data: throughputData,
+    error: throughputError,
+  } = useSpanMetricsSeries({
+    search: MutableSearch.fromQueryObject(filters),
+    yAxis: ['spm()'],
+    enabled: Boolean(groupId),
+    referrer: 'api.starfish.span-summary-page-charts',
+  });
+
+  const {
+    isLoading: isAvgDurationDataLoading,
+    data: avgDurationData,
+    error: avgDurationError,
+  } = useSpanMetricsSeries({
+    search: MutableSearch.fromQueryObject(filters),
+    yAxis: [`avg(${SpanMetricsField.SPAN_SELF_TIME})`],
+    enabled: Boolean(groupId),
+    referrer: 'api.starfish.span-summary-page-charts',
+  });
```

Comment:
this can be a single query

Reply:
I could combine them, but it feels noticeably slower when I fetch for both in the same query 🤔

Reply:
@shruthilayaj  I'm gonna opt to leave them as separate queries so they can run in parallel, which results in the charts feeling like they load faster. I talked to George about this and this was the reason why he built other pages with this pattern

Reply:
@0Calories also, didn't we decide to use duration instead of self time for the summary pages?